libraries/api/src/middleware/errors.js

```diff
@@ -1,5 +1,8 @@
 import { reportError } from '../error-reply.js';
 
+/**
+ * @typedef {Record<string, number>} ErrorCodes
```

Comment:
This just defines ErrorCodes, but doesn't assign it to the variable.

```suggestion
 * @type {Record<string, number>}
```

Reply:
oh nice catch, thank you!
I think I added this to import from another file but forget to define it as type here as well. Fixed